hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1480,6 +1480,16 @@
       run_opts: ["+use_otp_image=LcStProd"]
       reseed: 10
     }
+    //--------------------------------------------
+    //
+    {
+      name: chip_sw_power_idle_load
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["//sw/device/tests:chip_power_idle_load:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+en_scb_ping_chk=0"] // this is to disable ping-timeout check in cip_base_scoreboard, as it seems to be flawed.
```

Comment:
hmmm.... well that's not good. Do you mean when you enable ping checks you actually see failures? Or that alerts fire unexpectedly? This would be worth looking at.

Reply:
@cindychip is this ping check working as expected, or is this a known issue?

Reply:
I will need to look at the waves. I think it might be a SCB or alert_esc_agent issue, because this check is mainly designed for block level.
Let me dump the waves today :)
If it is a DV issue, we can add a TODO to fix it.

Reply:
I will open an issue.